web/controllers/workflow_controller.ex

```diff
@@ -10,7 +10,7 @@ defmodule Designator.WorkflowController do
     limit = get_integer_param(params, "limit", 5)
 
     subjects = Designator.Selection.select(strategy, workflow_id, user_id, limit)
-    Logger.metadata(%{selected_ids: Enum.join(subjects, ",")})
+    Logger.metadata([selected_ids: Enum.join(subjects, ",")])
```

Comment:
Is this a hash / dict object defined by `[]` ?

Reply:
Keyword list: https://elixir-lang.org/crash-course.html#keyword-list

Apparently it used to accept a Map too, but I guess that wasn't actually part of the official API since it doesn't work anymore. I'm guessing they added a type specification to that method sometime between these versions.